### PR TITLE
#CX-17126 : update menu overlay to not toggle on clearing the input

### PR DESCRIPTION
**Pull request by NamanGoel20**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by NamanGoel20** [comment]
Consider a situation where an md-input is the trigger element for the md-menu-overlay. Here, I am trying to stop the menu overlay from getting toggled, when the "clear" on the input is clicked.
md-input would require "stopPropagationOnClear" and md-menu-overlay would require "noToggleOnClearInput"

File: web-components/src/components/menu-overlay/MenuOverlay.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ export namespace MenuOverlay {
     @property({ type: String }) ariaRole: Role = "menu";
     @property({ type: String }) ariaLabel = "";
     @property({ type: Boolean, attribute: "is-date-picker" }) isDatePicker = false;
+    @property({ type: Boolean, attribute: "noToggleOnClearInput" }) noToggleOnClearInput = false; // This will prevent the overlay from toggling when the input field is cleared
```
Comment (mruane1) [inline web-components/src/components/menu-overlay/MenuOverlay.ts:76]: Very minor. noToggleOnClearInput being set to false implies there is toggle on clearing. Maybe rename to just
toggleOnClearInput = false;

Comment (NamanGoel20) [reply web-components/src/components/menu-overlay/MenuOverlay.ts:76]: Sure, will do!

**Comment by NamanGoel20** [comment]
Changes not required

**Pull request closed without merging**